### PR TITLE
Added default to order by

### PR DESCRIPTION
**Pull request by berenda-andrey**

### CODE REVIEW

File: tests/pagination/test_page_position_argument_resolver.py [hunk 1]
```diff
@@ -42,15 +42,14 @@ def func(arg1: argument_type):
     ('limit=10&offset=20&order_by=-id,name', PagePosition(10, 20, Sort.by('id').desc().and_(Sort.by('name')))),
     ('order_by= x', PagePosition(None, None, Sort.by(' x'))),
     ('order_by=- x', PagePosition(None, None, Sort.by(' x').desc())),
-    ('order_by= -x', PagePosition(None, None, Sort.by(' -x'))),
```
Comment (berenda-andrey) [inline tests/pagination/test_page_position_argument_resolver.py]: вроде не валидное значение

File: winter/pagination/page_position_argument_resolver.py [hunk 2]
```diff
@@ -109,3 +112,18 @@ def _parse_order(self, field: str, allowed_order_by_fields: typing.FrozenSet[str
 
         direction = SortDirection.DESC if is_desc else SortDirection.ASC
         return Order(direction=direction, field=field)
+
+    def _get_default_sort(self, argument: ComponentMethodArgument):
```
Comment (mofr) [inline winter/pagination/page_position_argument_resolver.py]: Add return type hint.

File: winter/pagination/page_position_argument_resolver.py [hunk 3]
```diff
@@ -109,3 +112,18 @@ def _parse_order(self, field: str, allowed_order_by_fields: typing.FrozenSet[str
...
+
+        if argument in self._default_sorts:
+            return self._default_sorts.get(argument)
+
+        order_by_annotation = argument.method.annotations.get_one_or_none(OrderByAnnotation)
+        if order_by_annotation is not None and order_by_annotation.default:
+            sort_parts = order_by_annotation.default.split(',')
+            orders = [self._parse_order(sort_part, order_by_annotation.allowed_fields) for sort_part in sort_parts]
+            sort = Sort(*orders)
```
Comment (mofr) [inline winter/pagination/page_position_argument_resolver.py]: 3 rows are duplicated with `_parse_sort_properties`

File: winter/pagination/sort/order_by_annotation.py [hunk 4]
```diff
@@ -2,16 +2,8 @@
...
 class OrderByAnnotation:
     allowed_fields: typing.FrozenSet[str]
-
-
-def get_allowed_order_by_fields(method: ComponentMethod) -> typing.FrozenSet[str]:
-    order_by_annotation = method.annotations.get_one_or_none(OrderByAnnotation)
-    if order_by_annotation is None:
-        return frozenset()
-    return order_by_annotation.allowed_fields
+    default: typing.Optional[str]
```
Comment (mofr) [inline winter/pagination/sort/order_by_annotation.py]: I was expecting `default_sort: Optional[Sort]` here, does it make sense?

File: winter/pagination/sort/order_by.py [hunk 5]
```diff
@@ -4,7 +4,11 @@
 from ...core.annotation_decorator import annotate_method
 
 
-def order_by(allowed_fields: typing.Iterable[str]):
+def order_by(allowed_fields: typing.Iterable[str], default: typing.Optional[str] = None):
     allowed_fields = frozenset(allowed_fields)
-    annotation = OrderByAnnotation(allowed_fields)
+    if default is not None:
+        assert all(field in allowed_fields for field in default.split(',')), (
```
Comment (mofr) [inline winter/pagination/sort/order_by.py]: Looks like a third duplicate of the parsing logic, can we have a function `parse_sort(str) -> Sort` and use it everywhere?

File: winter/pagination/page_position_argument_inspector.py [hunk 6]
```diff
@@ -39,12 +39,16 @@ def inspect_parameters(self, route: 'Route') -> List[openapi.Parameter]:
...
+        order_by_annotation = route.method.annotations.get_one_or_none(OrderByAnnotation)
+        if order_by_annotation:
+            allowed_order_by_fields = ','.join(map(str, order_by_annotation.allowed_fields))
+            default_part = f' Default is "{order_by_annotation.default}"' if order_by_annotation.default else ''
             order_by_parameter = openapi.Parameter(
                 name=self._page_position_argument_resolver.order_by_name,
-                description=f'Comma separated order by fields. Allowed fields: {allowed_order_by_fields}',
+                description=(
+                    f'Comma separated order by fields. '
+                    f'Allowed fields: {allowed_order_by_fields}.{default_part}'
```
Comment (mofr) [inline winter/pagination/page_position_argument_inspector.py]: Why default_part is in the description and not in openapi.Parameter default property?

File: winter/pagination/page_position_argument_resolver.py [hunk 7]
```diff
@@ -33,6 +33,8 @@ def __init__(
         self.order_by_name = order_by_name
         self.limits = Limits(default=None, maximum=None, redirect_to_default=False)
 
+        self._default_sorts = {}
```
Comment (mofr) [inline winter/pagination/page_position_argument_resolver.py]: It's not used anymore

**Pull request merged**